### PR TITLE
kubernetes: Reverse zone fall through

### PR DESCRIPTION
**Pull request by chrisohaver**
- Add reverse zone fall through to config.
- Move to flag based parameters (the string of all optional parameters was getting clumsy).

### CODE REVIEW

File: kubernetes/deploy.sh [hunk 1]
```diff
@@ -2,23 +2,51 @@
...
-
-if [[ -z $SERVICE_CIDR ]]; then
-	echo "Usage: $0 SERVICE-CIDR [ POD-CIDR ] [ DNS-IP ] [ CLUSTER-DOMAIN ] [ YAML-TEMPLATE ]"
-	exit 1
-fi
+show_help () {
+cat << USAGE
+usage: $0 [ -r REVERSE-CIDR ] [ -i DNS-IP ] [ -d CLUSTER-DOMAIN ] [ -t YAML-TEMPLATE ]
+
+    -r : Define a reverse zone for the given CIDR. You may specifcy this option more
```
Comment (fturib) [inline kubernetes/deploy.sh]: There is a sample of deploy.sh in the README.md.
Need to update I think.

Comment (chrisohaver) [reply kubernetes/deploy.sh]: thanks - updated readme

**Comment by chrisohaver** [comment]
test was still running 1.0.1, hence failures. rebased and re-running /integration

**Comment by chrisohaver** [comment]
closes #48

**Pull request merged**